src/components/app/app.test.tsx

```diff
@@ -1,12 +1,18 @@
-import {shallow} from 'enzyme';
-import React from 'react';
+/**
+ * @jest-environment jsdom
```

Comment:
That seems like a noise to me. I know that the reason for not putting this into `package.json` is in case we have other files, but frankly, I don't think it's much of an issue bearing in mind this is the frontend project and we should barely have different files that will be tested with Jest.

Screw it. Let's move it to `package.json` to reduce code smell.